crates/nu-cli/src/commands/str_/to_decimal.rs

```diff
@@ -46,7 +46,7 @@ impl WholeStreamCommand for SubCommand {
     fn examples(&self) -> Vec<Example> {
         vec![Example {
             description: "Convert to float",
```

Comment:
and this `float`

Reply:
Looks like there might be 3 total in the file

Reply:
Alright
on it